app/pages/lab/social-links-editor.spec.js

```diff
@@ -2,8 +2,18 @@ import React from 'react';
 import assert from 'assert';
 import { shallow, mount } from 'enzyme';
 import sinon from 'sinon';
+import apiClient from 'panoptes-client/lib/api-client';
 import SocialLinksEditor from './social-links-editor';
 
+function mockPanoptesResource(type, options) {
+  const resource = apiClient.type(type).create(options);
+  apiClient._typesCache = {};
```

Comment:
Non-blocking comment, but there is an undocumented `uncache()` class method on Resource: https://github.com/zooniverse/json-api-client/blob/c6dd8e406c2fcb02db1a84f85c6b7d85fe068179/src/resource.coffee#L86-L92

I have no idea if it does what is needed for these tests to work though.

Reply:
From the code, it looks like it removes the current resource from the client cache, so `resource.uncache()` probably would work here too.